### ISSUE TITLE
WCAG: 4.1.2 - list are missing  "ARIA input field"

### ISSUE THREAD

**Issue by JoaoEnesGoncalves** [lumada-design/hv-uikit-react#778]
**Describe the bug**
the embeded listbox are missing rule 'ensures every ARIA input field has an accessible name' 
- https://dequeuniversity.com/rules/axe/3.4/aria-input-field-name?application=axeAPI

**affects composed components:**
  ❌  dropdown
  ❌  verticalNavigation
  ❌ searchBox (suggestions list)

**To Reproduce**
run
```
node C:\Git\hv-uikit-react\automation\robot\storybook\_resources/accessibility.js --url=http://localhost:9002/iframe.html?id=coredropdownmenu--dropdownmenu1 --standard=WCAG2AA --element="#root>div.Component-content-4" --actions="click element div[class|='DropDownMenu'][role='button']"
Screenshots
```

**Screenshots**
```
{
  documentTitle: 'Storybook',
  pageUrl: 'http://localhost:9002/iframe.html?id=coredropdownmenu--dropdownmenu1',
  issues: [{
      code: 'aria-input-field-name',
      type: 'error',
      typeCode: 1,
      message: 'ARIA input fields have an accessible name (https://dequeuniversity.com/rules/axe/3.4/aria-input-field-name?application=axeAPI)',
      context: '<ul id="" class="List-root-156" role="listbox"><li role="option" aria-selected...</ul>',
      selector: '#root > div:nth-child(2) > div > div > div > div > div:nth-child(2) > div > ul',
      runner: 'axe',
      runnerExtras: [Object]
    }
  ]
}
```
```
{
  documentTitle: 'Storybook',
  pageUrl: 'http://localhost:9002/iframe.html?id=coreverticalnavigation--verticalnavigation1',
  issues: [{
    code: 'aria-input-field-name',
    type: 'error',
    typeCode: 1,
    message: 'ARIA input fields have an accessible name (https://dequeuniversity.com/rules/axe/3.4/aria-input-field-name?application=axeAPI)',
    context: '<ul id="" class="List-root-157" role="listbox"><li role="option" aria-selected...</ul>',
    selector: '#root > div:nth-child(2) > div > div > div > div > div > div > div > ul',
    runner: 'axe',
    runnerExtras: [Object]
  }]
}
```
